src/drivers/dw/dma.c

```diff
@@ -550,8 +550,7 @@ static int dw_dma_set_config(struct dma_chan_data *channel,
 		if (dw_chan->lli)
 			rfree(dw_chan->lli);
 
-		dw_chan->lli = rmalloc(SOF_MEM_ZONE_SYS_RUNTIME, 0,
-				       SOF_MEM_CAPS_RAM | SOF_MEM_CAPS_DMA,
+		dw_chan->lli = rballoc(0, SOF_MEM_CAPS_RAM | SOF_MEM_CAPS_DMA,
```

Comment:
Since you claim the problem is about alignment, why not `rballoc_align` with a specific value that ought to solve it? Even if you pass it PLATFORM_DCACHE_ALIGN which thus makes it equivalent, I think it's better to be explicit in this particular case.

Reply:
@paulstelian97 It depends on how `rballoc()` is defined. If it is a part of its definition, that returned memory is always cache-line size aligned, then I think the code is fine as is. If however it's just a kind of a hidden side-effect, that might change in the future, then yes, it would be better to change this. I'm fine either way. Let's wait for a couple of hours, if there are no opinions one way or another, if the CI "failure" doesn't mean this is breaking functionality, I can change it, sure.

Reply:
Oh it's completely equivalent, `rballoc(...)` is precisely `rballoc_aligned(..., PLATFORM_DCACHE_ALIGN)`. However I wanted to see the alignment being explicitly specified for coding style points.